tests/parametric/test_dynamic_configuration.py

```diff
@@ -684,7 +685,8 @@ def test_trace_sampling_rules_override_rate(self, library_env, test_agent, test_
             }
         ],
     )
-    def test_trace_sampling_rules_with_tags(self, library_env, test_agent, test_library):
+    @bug(context.library == "cpp")
```

Comment:
add the cause

Reply:
> ~~add the cause~~
Could you add the cause? FTFY. Yes, I can.